modules/manual-configuration-image-registry-ecr.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ecr_{context}"]
+= Manually configuring Amazon Elastic Container Registry
+
+You can integrate {product-title} with Amazon Elastic Container Registry.
+
+.Prerequisites
+* You need an *Access Key ID* and a *Secret Access Key*. Alternatively, you can use a node-level IAM proxy such as `kiam` or `kube2iam`.
+* The access key must have read access to ECR.
+See https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/[How do I create an AWS access key?]  for more information.
+* If you are running {product-title} in Amazon Elastic Kubernetes Service (EKS) and want to integrate with an ECR from a separate Amazon account, you must first set a repository policy statement in your ECR.
+Follow the instructions at https://docs.aws.amazon.com/AmazonECR/latest/userguide/set-repository-policy.html[Setting a Repository Policy Statement]  and for *Actions*, choose the following scopes of the Amazon ECR API operations:
```

Comment:
Setting a Repository Policy Statement -> Setting a repository policy statement (per the linked page heading)